src/test/bash/asset_create_with_upload.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
Yes, those are for generating the responses needed for the upload module, directly from contentful. There are used for generating the ground truth of my canned tests.